tests/lib/rules/no-test-id-queries.test.ts

```diff
@@ -0,0 +1,78 @@
+import rule, { RULE_NAME } from '../../../lib/rules/no-test-id-queries';
+import { createRuleTester } from '../test-utils';
+
+const ruleTester = createRuleTester();
+
+const SUPPORTED_TESTING_FRAMEWORKS = [
+	'@testing-library/dom',
+	'@testing-library/angular',
+	'@testing-library/react',
+	'@testing-library/vue',
+	'@marko/testing-library',
+];
+
+const QUERIES = [
+	'getByTestId',
+	'queryByTestId',
+	'getAllByTestId',
+	'queryAllByTestId',
+	'findByTestId',
+	'findAllByTestId',
+];
+
+ruleTester.run(RULE_NAME, rule, {
+	valid: [
```

Comment:
suggestion (blocking): Please add another valid test that includes calls to methods containing `*TestId` at the end but not matching the regex reported by the rule (e.g. `caculateTestId`, `getTestId`, `searchByTestId`)

Reply:
Done